docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -6,17 +6,19 @@
 
 
 Published Mar 2023  
-V1.3.16 Edition
+V8.3.4 Edition

```

Comment:
This is the iso version. Shoudn't it be like V1.5.1

Reply:
Based on our discussion, this version is align with product version